sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -254,6 +254,7 @@ def __init__(self, name, transforms,
         self.transforms = transforms
         self.downstream_side_inputs = downstream_side_inputs
         self.must_follow = must_follow
+        self.timers = []
```

Comment:
It would be a lot easier to follow if this class docstring had comments on the expected types of these arguments / attributes.